### PR TITLE
Added 101-1vm-2nics-2subnets-1vnet

### PR DESCRIPTION
**Pull request by kpatnayakuni**
I have picked up an ARM template '101-1vm-2nics-2subnets-1vnet' from azure-quickstart-templates and created terraform configuration for the same.

### CODE REVIEW

**Review by grayzu — CHANGES_REQUESTED** [review]
Looks great, @kpatnayakuni!! Thanks for the PR. 
Can you please update the syntax to 0.12 syntax by removing all of the unnecessary "${ , }" around the variables? Otherwise it is good to go.

**Comment by kpatnayakuni** [comment]
Thanks, @grayzu for reviewing my PR, I have updated the configuration to v0.12.16 and request you to review again. Thank you.

File: quickstart/101-1vm-2nics-2subnets-1vnet/readme.md [hunk 1]
```diff
@@ -0,0 +1,402 @@
...
+| virtual_machine_size | Virtual machine size (has to be at least the size of Standard_A3 to support 2 NICs) |
+| admin_username | Default Admin username |
+| admin_password | Default Admin password |
+| storage_account_type | Storage Account type for the VM and VM diagnostic storage |
+
+## Resources
+| Resource Type | Description |
+|-|-|
+| azurerm_resource_group | The resource group for all the resources are deployed into |
+| zurerm_virtual_network | One Virtual Network |
```
Comment (grayzu) [inline quickstart/101-1vm-2nics-2subnets-1vnet/readme.md:19]: minor typo. Missing "a"

File: quickstart/101-1vm-2nics-2subnets-1vnet/variables.tf [hunk 2]
```diff
@@ -0,0 +1,35 @@
+provider "azurerm" {
+
```
Comment (grayzu) [inline quickstart/101-1vm-2nics-2subnets-1vnet/variables.tf:2]: Should add the version here for best practices.

**Review by grayzu — APPROVED** [review]
Thanks @kpatnayakuni !

I commented on few minor tweaks that would be great but approving regardless.

**Pull request closed without merging**